hw/top_earlgrey/dv/env/chip_env.core

```diff
@@ -7,6 +7,7 @@ description: "CHIP DV UVM environmnt"
 filesets:
   files_dv:
     depend:
+      - lowrisc:dv:spi_host_sva
```

Comment:
Nit: I'd probably suggest moving this around so that stuff stays in alphabetical order.